contracts/contracts/strategies/MorphoAaveStrategy.sol

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity ^0.8.0;
+
+/**
+ * @title OUSD Morpho Aave Strategy
+ * @notice Investment strategy for investing stablecoins via Morpho (Aave)
+ * @author Origin Protocol Inc
+ */
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20, InitializableAbstractStrategy } from "../utils/InitializableAbstractStrategy.sol";
+import { IMorpho } from "../interfaces/morpho/IMorpho.sol";
+import { ILens } from "../interfaces/morpho/ILens.sol";
+import { StableMath } from "../utils/StableMath.sol";
+import "../utils/Helpers.sol";
```

Comment:
I don't think it's used

Reply:
@shahthepro. We should go ahead and remove this line if it's not used. Will keep the codebase clean.

Reply:
Sure, will remove and redeploy the contracts today